apps/challenges/models.py

```diff
@@ -185,18 +179,25 @@ def is_active(self):
         return False
 
 
-signals.post_save.connect(
-    model_field_name(field_name="evaluation_script")(
-        restart_workers_signal_callback
-    ),
-    sender=Challenge,
-    weak=False,
-)
+def post_save_connect(field_name, sender):
+    import challenges.aws_utils as aws
+
+    signals.post_save.connect(
+        model_field_name(field_name=field_name)(
+            aws.restart_workers_signal_callback
+        ),
+        sender=sender,
+        weak=False,
+    )
+
+
+post_save_connect("evaluation_script", Challenge)
```

Comment:
@burnerlee can we move `post_save_connect("test_annotation", ChallengePhase)` from line 333 to line 195.

Reply:
`ChallengePhase` model is defined later in the file. Hence to move this callback up, I would also need to move that model class upwards in the file accordingly

Reply:
@burnerlee you can move the `post_save_connect` callback from line 194 to line 332 then. It would be good to have all the post save callbacks at same place